website/blog/2023-02-24-Deploy-databend-on-minio.md

```diff
@@ -0,0 +1,211 @@
+
+
+### Databend support deployment environments
+
+| Environments| Databend |
+| ----------- | -------- |
+| AWS S3      | Yes     |
+| Google GCS  | Yes     |
+| Azure Blob  | Yes     |
+| Aliyun OSS  | Yes    |
+| Tencent COS | Yes     |
+| Huawei OBS  | Yes     |
+| MinIO       | Yes     |
+| Ceph        | Yes     |
+| Wasabi      | Yes     |
+| SeaweedFS   | Yes     |
+| QingStor    | Yes     |
+
+Detailed reference: https://databend.rs/doc/deploy/deploying-databend
+
+## Databend Single-machine deployment
+
+> In essence, the minio environment is relatively easy to set up. However, if you use in production, you can use the MinIO Cloud or AWS S3 environment to reduce hassle.
+
+The following uses the MinIO + Databend standalone deployment in Linux of x64 as an example:
+
+| Software | path           | port                                                |
+| -------- | -------------- | --------------------------------------------------- |
+| minio    | /data/minio    | 9900                                                |
+| databend | /data/databend | mysql: 3307 <br/> http:  8000 <br/>Clickhouse http:  8124 |
+
+### Minio Deployment
+
+MinIO Homepage: https://min.io/
+
+Download MinIO from the official website and start by running the following commands:
+
+```Bash
+cd /data 
+mkdir minio
+cd minio 
+wget https://dl.min.io/server/minio/release/linux-amd64/minio
+export MINIO_ROOT_USER=minioadmin
+export MINIO_ROOT_PASSWORD=minioadmin
+./minio server ./data./minio server --address :9900 ./data
+```
+Use the MinIO Admin web interface to create a "databend" bucket.
+
+![](../static/img/blog/databend-minio-beginner-01-2.png)
+
+After creating the bucket, the MinIO deployment is complete.
+
+### Download and Install Databend
+
+| type                                                         | os            |
+| ------------------------------------------------------------ | ------------- |
+| - databend-version-nightly-x86_64-unknown-linux-musl.tar.gz  | Linux for x64 |
+| - databend-version-nightly-aarch64-apple-darwin.tar.gz       | MacOS of M1   |
+| - databend-version-nightly-aarch64-unknown-linux-musl.tar.gz | Linux of Arm  |
+| - databend-version-nightly-x86_64-apple-darwin.tar.gz        | MacOS of X64  |
+| - source code                                                | src           |
+
+We can download the musl package for Linux from the Databend open-source project repo a: https://github.com/datafuselabs/databend/tags
+
+
+Generally, we can download the musl package for Linux. Note the difference between arm and x86_64 platforms.
+
+Download: https://github.com/datafuselabs/databend/tags
+
+For version selection, it is recommended to download the latest version every day. Databend is fast to develop and many new features are merged quickly.For example, if you're running Linux on an x86_64 machine, you can download version v0.9.49 like this:
```

Comment:
The latest version is always recommended as Databend develops and merges new features on a daily basis. For example, if you're running Linux on an x86_64 machine, you can download version v0.9.49 like this: